app/client/ui/OpenAccessibilityModal.ts

```diff
@@ -0,0 +1,184 @@
+            }, 100);
+          }),
+          testId('high-contrast-theme-button'),
+        )),
+      );
+    }),
+    dom('p', t("To see other available themes, go to your {{profileSettingsLink}}.", {
+      profileSettingsLink: cssLink(
+        // close the modal when clicking the profile settings link, in case we already are on the profile settings page.
+        dom.on('click', () => ctl.close()),
+        urlState().setLinkUrl({account: 'account'}),
+        t("profile settings"),
+      )
+    }))
+  );
+};
+
+const keyboardSection = () => {
+  const nextRegionShortcut = dom('span', getCssKeys(allCommands.nextRegion.humanKeys));
+  const prevRegionShortcut = dom('span', getCssKeys(allCommands.prevRegion.humanKeys));
+  const creatorPanelShortcut = dom('span', getCssKeys(allCommands.creatorPanel.humanKeys));
+  const shortcutsModal = dom('span', getCssKeys(allCommands.shortcuts.humanKeys));
+  const accessibilityModal = dom('span', getCssKeys(allCommands.accessibility.humanKeys));
+  return cssSection(
+    cssModalSubheading(t("Keyboard navigation"), {role: "heading", "aria-level": 2}),
+    dom('p', t("On a document page, keyboard navigation is first locked on the current widget.")),
+    dom('p', t("Focus on other parts of the user interface using the following shortcuts:")),
+    dom('ul',
+      cssShortcutRow(t("{{nextRegionShortcut}} Focus on the next region", {nextRegionShortcut})),
+      cssShortcutRow(t("{{prevRegionShortcut}} Focus on the previous region", {prevRegionShortcut})),
+      cssShortcutRow(t("{{creatorPanelShortcut}} Focus to and from the creator panel", {creatorPanelShortcut})),
+    ),
+    dom('p', t("\"Regions\" are what we call the different parts of the user interface:")),
+    dom('ul',
+      dom('li', t("The left panel, home of the main navigation.")),
+      dom('li', t("The top panel, or the document header.")),
+      dom('li', inlineMarkdown(t(
+        "On document pages, each [widget]({{supportPageUrl}}) is a region that can receive focus.",
+        {supportPageUrl: commonUrls.helpWidgets}
+      ))),
+      dom('li', t("On non-document pages, the main content area is a region.")),
+      dom('li', t("Finally, the right panel – or the creator panel – is only available \
+through its own shortcut and is not included in the next and previous region cycle.")),
+    ),
+    cssModalSubheading(t("Other important keyboard shortcuts"), {role: "heading", "aria-level": 2}),
+    dom('ul',
+      cssShortcutRow(t("{{shortcutsModal}} Show the complete list of keyboard shortcuts", {shortcutsModal})),
+      cssShortcutRow(t("{{accessibilityModal}} Show the accessibility options (this modal)", {accessibilityModal})),
+    ),
+  );
+};
+
+const getCssKeys = (keys: string[]) => {
+  return keys.map((k, i) => i === keys.length - 1 ? cssKey(k) : [cssKey(k), t(" or ")]);
+};
+
+const cssSection = styled('div', `
+  margin-bottom: 32px;
+  &:last-child {
+    margin-bottom: 0;
+  }
+  ${nestedLinkStyles}
```

Comment:
Rather than copy css rules for links into other css rules, I think it's better to use css classes. One way is to have `cssNestedLinks`; then this could be defined as `cssSection = styled(cssNestedLinks, "...")` (it would just be an element with both classes). Or when creating the element, you could add the extra class manually, like `cssSection(cssNestedLinks.cls(''), ...content...)`